src/amo/pages/Guides/index.js

```diff
@@ -0,0 +1,193 @@
+           stolen. Fortunately, extensions can help fortify your online privacy
+           and security.`,
+        ),
+        icon: 'stop-hand',
+        sections: [
+          // Bitwarden free password manager
+          {
+            addonGuid: '{446900e4-71c2-419f-a6a7-df9c091e268b}',
+            header: i18n.gettext('Create and manage strong passwords'),
+            description: i18n.gettext(
+              `Password managers can help you create secure passwords, store your
+               passwords (safely) in one place, and give you easy access to your
+               login credentials wherever you are.`,
+            ),
+            addonCustomText: i18n.gettext(
+              `Fully encrypted password protection. Store your data securely and
+               access logins across devices.`,
+            ),
+            exploreMore: i18n.gettext(
+              'Explore more %(linkStart)spassword manager%(linkEnd)s staff picks.',
+            ),
+            exploreUrl: '/collections/mozilla/password-managers/',
+          },
+        ],
+      };
+    default:
+      return null;
+  }
+};
+
+export class GuidesBase extends React.Component<InternalProps> {
+  constructor(props: InternalProps) {
+    super(props);
+
+    const { errorHandler, i18n, match } = this.props;
+    const { slug } = match.params;
+    const content = getContent(slug, i18n);
+
+    if (!content) {
+      return;
+    }
+
+    const guids = content.sections.map((section) => section.addonGuid);
+
+    this.props.dispatch(
+      fetchGuidesAddons({
+        guids,
+        errorHandlerId: errorHandler.id,
+      }),
+    );
+  }
+
+  getGuidesSections = (
+    sections: Array<SectionsType>,
+  ): React.ChildrenArray<React.Node> => {
+    const { clientApp, i18n, lang } = this.props;
+
+    return sections.map((section) => {
+      const exploreMoreLink = i18n.sprintf(section.exploreMore, {
+        linkStart: `<a class="Guides-section-explore-more-link" href="/${lang}/${clientApp}${
+          section.exploreUrl
+        }">`,
+        linkEnd: '</a>',
+      });
+
+      return (
+        <div className="Guides-section" key={section.exploreUrl}>
+          <h2 className="Guides-section-title">{section.header}</h2>
+          <p className="Guides-section-description">{section.description}</p>
+
+          <div
+            className="Guides-section-explore-more"
+            // eslint-disable-next-line react/no-danger
+            dangerouslySetInnerHTML={sanitizeHTML(exploreMoreLink, ['a'])}
+          />
+        </div>
+      );
+    });
+  };
+
+  render() {
+    const { i18n, match } = this.props;
+    const { slug } = match.params;
+    const content = getContent(slug, i18n);
+
+    if (!content) {
+      return <NotFound />;
+    }
+
+    const { introText, icon, title, sections } = content;
+
+    return (
+      <div className="Guides-page">
+        <Helmet>
+          <title>{title}</title>
```

Comment:
We need a test case for that. Search for "renders an HTML title" in the test files, you'll find examples.

Reply:
yep i did this but I've added as part of the "renders a Guide Page" test case..I can extract this out too tho..